Assets/MixedRealityToolkit/_Core/Interfaces/InputSystem/Handlers/IMixedReality3DoFInputHandler.cs

```diff
@@ -6,14 +6,20 @@
 namespace Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem.Handlers
 {
     /// <summary>
-    /// Interface to implement for 3 Degree of Freedom input.
+    /// Interface to implement input for 3 Degrees of Freedom.
     /// </summary>
     public interface IMixedReality3DoFInputHandler : IMixedRealityInputHandler
     {
         /// <summary>
-        /// 3 Degree of Freedom input update.
+        /// Raised when the input source's position has changed.
         /// </summary>
         /// <param name="eventData"></param>
-        void On3DoFInputChanged(ThreeDoFInputEventData eventData);
+        void OnPositionChanged(PositionInputEventData eventData);
```

Comment:
3DoF is position or orientation. This feels more like a 6DoF handler. Can we rename this method as On3DoFDataChanged and remove OnRotationChanged? I think  we may want to add a hint to the eventData object.

Reply:
Please ignore the above... in taking offline with @keveleigh, I get the value of the two methods.